samples/BlobStorageProcessor/Program.cs

```diff
@@ -58,7 +58,7 @@ public static async Task Main(string[] args)
             // Log in to Waives
             await WaivesApi.Login("clientId", "clientSecret");
 
-            var blobStorage = BlobStorageDocumentSource.Create(blobs);
+            var blobStorage = new EnumerableDocumentSource(blobs.Select(b => new BlobStorageDocument(b)));
             var writer = CreateResultWriter(options["--output"].ToString());
             var pipeline = WaivesApi.CreatePipeline()
                 .WithDocumentsFrom(blobStorage)
```

Comment:
Let's come back to this one. I left it as `blobStorage` so that the pipeline definition read more fluently: "a pipeline with documents from blob storage".